src/facebook_example/bot.clj

```diff
@@ -28,6 +29,17 @@
       ; If no rules apply echo the user's message-text input
       :else (reaction/echo message-text))))
 
+(defn on-quick-reply [event]
+  (println "on-quickreply event:")
+  (println event)
+  (let [sender-id (get-in event [:sender :id])
+        quick-reply (get-in event [:message :quick_reply])
+        quick-reply-payload (:payload quick-reply)]
+    (cond
+      (= quick-reply-payload "CLOJURE") (reaction/send-clojure-docs)
```

Comment:
As far as I know this seems to be a convention for payloads even in the Messenger docs.